swagger_to_sdk_config.json

```diff
@@ -1,16 +1,20 @@
 {
   "meta": {
     "after_scripts": [
-      "gofmt -w ./services/"
+      "gofmt -w ./services/",
+      "go get -u github.com/Azure/azure-sdk-for-go/tools/profileBuilder",
+      "profileBuilder -s list -l ./profiles/2017-03-09/defintion.txt -name 2017-03-09",
+      "profileBuilder -s preview -name preview",
+      "profileBuilder -s latest -name latest"
     ],
     "autorest_options": {
       "use": "@microsoft.azure/autorest.go@preview",
       "go": "",
       "verbose": "",
       "sdkrel:go-sdk-folder": ".",
       "multiapi": "",
-      "package-version": "v12.2.1-beta",
-      "user-agent": "Azure-SDK-For-Go/v12.2.1-beta services"
+      "package-version": "nightly",
+      "user-agent": "Azure-SDK-For-Go/nightly services"
```

Comment:
Should we also add the date, or is it too much?

Reply:
That is the idea yes however I got the impression that more tooling work needs to happen to support this.

Reply:
If you want a date, I can probably accept a C pattern (strftime) and use "now()" to fill it
https://docs.python.org/3/library/datetime.html#strftime-and-strptime-behavior

Tell me if it helps

Reply:
Yes I think that would be great!  It can work for both package-version and user-agent?

Reply:
It will work for any option. I will like "sdkrel" prefix and add prefix to explain there is date parameter to expand. Something like (pseudo-code):
```json
"date:package-version": "{%Y-%M}"
```
Let me think about it, that's totally doable